packages/styleguide/src/components/Typography/Editorial.tsx

```diff
@@ -196,8 +196,8 @@ const format = css({
 type FormatProps = {
   children: React.ReactNode
   color?: string
-  attributes?: React.HTMLAttributes<HTMLParagraphElement>
-} & React.HTMLAttributes<HTMLParagraphElement>
+  attributes?: React.ComponentProps<'p'>
```

Comment:
Tired of making more suggestions 😅, just find and replace the rest of `ComponentProps`